data_steward/tools/generate_ext_tables.py

```diff
@@ -5,11 +5,10 @@
 import constants.bq_utils as bq_consts
 import constants.cdr_cleaner.clean_cdr as cdr_consts
 from resources import fields_for, MAPPING_TABLES
+from common import JINJA_ENV
 
 LOGGER = logging.getLogger("__name__")
 
-LOGGER = logging.getLogger(__name__)
-
```

Comment:
This should be left in.  The `__name__` variable is a special python variable used to name each module.  This is setting the name of the python logger to the module name.  So you see logs like this.
```
2020-11-03 12:18:03,439 - INFO - __main__ - using dynamic extension table schema for table: procedure_occurrence_ext
```